tools/release-init.sh

```diff
@@ -18,9 +18,9 @@ git push -u origin release/${realm_version}
 git checkout -b prepare-$realm_version
 
 # update dependencies.list
-sed -i "s/^VERSION.*/VERSION=${realm_version}/" ${project_dir}/dependencies.list
+sed -i -e "s/^VERSION.*/VERSION=${realm_version}/" ${project_dir}/dependencies.list
```

Comment:
I don't think this does quite what you want. With BSD sed, used on Mac, `-i -e` will result in a backup file being created with a `-e` extension (i.e., `dependencies.list-e`). To edit in place, without a backup file, you need to use `-i ''` with BSD sed and `-i` by itself with GNU sed.

It might be simpler to explicitly specify a backup extension and then clean up the backup files.